woodwork/logical_types.py

```diff
@@ -121,7 +121,8 @@ def validate(self, series, return_invalid_values=True):
         Returns:
             Series: If return_invalid_values is True, returns invalid age values.
         """
-        super().validate(series, return_invalid_values)
+        # Comment this out as so far only check valid_dtype for Ordinal and LatLong type
+        # super().validate(series, return_invalid_values)
```

Comment:
Let's remove the comment and the commented out lines from the Logical Types that are no longer using `super()` to avoid clutter. Instead, if you'd like to put a comment indicating when `super().validate(series, return_invalid_values)` should be used within child classes, that can go within `LogicalType.validate` itself.